pandas/io/common.py

```diff
@@ -2,6 +2,7 @@
 
 import bz2
 import codecs
+from collections.abc import Iterator
```

Comment:
this should be failing ci on Check for non-standard imports

log shows
```
##[error]pandas/io/stata.py:12:from collections.abc import Iterator
##[error]pandas/io/sas/sas7bdat.py:16:from collections.abc import Iterator
##[error]pandas/io/sas/sas_xport.py:10:from collections.abc import Iterator
##[error]pandas/io/common.py:5:from collections.abc import Iterator
##[error]pandas/io/parsers.py:6:from collections.abc import Iterator
##[error]pandas/io/json/_json.py:2:from collections.abc import Iterator
```

Reply:
changes to invgrep in #29915. maybe related. @datapythonista

Reply:
It's strange, because I tested that the exit codes were correct in that PR, but I agree the problem is likely to be introduced in that PR.

I'm travelling without my computer at the moment, and I can't test myself. But feel free to revert that PR if you identify it's the one causing the problem, and I'll redo it with the problem fixed later.

Reply:
what is the preferred way to do the import?

Reply:
What is the correct way to do the import? I'm getting this failure in my PR https://github.com/pandas-dev/pandas/pull/30151/checks?check_run_id=362567629

Reply:
I think the import is correct. Must be something wrong with the invgrep as noted by @simonjayhawkins

Reply:
@alimcmaster1 might have an idea here as well

Reply:
Sure from reading https://github.com/pandas-dev/pandas/pull/25957#pullrequestreview-221448898 where this check was implemented our preferred way to do the import is `from collections import abc`. Or `from collections import Iterator` 

I can update this now if helpful @mroeschke ?

Agree - seems to be something flakey with the invgrep on github actions since fails for me locally.

Reply:
That'd be great @alimcmaster1, thanks!